bigquery/samples/list_datasets_by_label.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def list_datasets_by_label(client):
+
+    # [START bigquery_list_datasets_by_label]
+    # TODO(developer): Import the client library.
+    # from google.cloud import bigquery
+
+    # TODO(developer): Construct a BigQuery client object.
+    # client = bigquery.Client()
+
+    label_filter = "labels.color:green"
+    datasets = list(client.list_datasets(filter=label_filter))
+
+    if datasets:
+        print("Datasets filtered by {}:".format(label_filter))
+        for dataset in datasets:  # API request(s)
```

Comment:
Since you've already called `list` on the return value, this actually doesn't make any additional API requests.

```suggestion
        for dataset in datasets:
```